packages/transactional-adapters/transactional-adapter-typeorm/test/transactional-adapter-typeorm.spec.ts

```diff
@@ -74,27 +71,17 @@ class UserService {
     })
     async transactionWithDecoratorWithOptions() {
         const r1 = await this.userRepository.createUser('James');
-        const r2 = await this.datasource
-            .getRepository(User)
-            .createQueryBuilder('user')
-            .where('user.id = :id', { id: r1.id })
-            .getOne();
```

Comment:
Ah, so this is a bummer.

Apparently TypeORM doesn't support connection pools and everything is done within a single connection (and I suspect that's why you removed these lines, because the tests kept breaking). This effectively means that TypeORM doesn't support transactions with `sqlite`.

In light of this unpleasant fact, I suggest you add back the compose file and run the test against postgres.

Reply:
Agree this is why also was better use postgres